packages/swing-store/src/swingStore.js

```diff
@@ -1073,7 +1073,7 @@ export async function importSwingStore(exporter, dirPath = null, options = {}) {
       transcriptInfo.startPos === 0 ||
         Fail`missing current snapshot for vat ${q(vatID)}`;
     } else {
-      snapshotInfo.endPos === transcriptInfo.startPos ||
+      snapshotInfo.endPos + 1 === transcriptInfo.startPos ||
```

Comment:
On deeper thought I'm reasonably convinced this is a mistake and that the old invariant check is the correct one.  The thing that's adding the snapshot entry into the transcript should be incrementing the `endPos` value the same as any other addition to the transcript.  After looking at the code that's adding these special entries, I'm puzzled by why that's not happening, but I think something is wrong there.

Reply:
I gave this some thoughts, and I believe this change is logically correct. However it highlights a potential problem with this PR
When we create a snapshot, we record the current transcript position `endPos`.
Now with this PR when the snapshot is created, we also insert a transcript entry with the snapshot details, then roll over the span.
It used to be that the new span started at that `endPos`, but now it starts at that `endPos + 1` because of the extra transcript entry.

Either we accept that difference (which has the nice property that the snapshot position is the position of the corresponding `snapshot-save` transcript entry), or we record the snapshot position as `endPos + 1` instead to keep the snapshotPos === next transcript span start.

Reply:
I just don't get this.  The `endPos` isn't part of the snapshot, so I don't see how the timing of the snapshot enters into the discussion at all.